ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js

```diff
@@ -0,0 +1,345 @@
+                        this._elements[key].push(hook);
+                    } else {
+                        this._elements[key] = hook;
+                    }
+                }
+            }
+        }
+
+        getClass() {
+            return Wizard.IS;
+        }
+
+        setFocus() {
+            this.setActive();
+        }
+
+        getWidget() {
+            return this.element.querySelector(Wizard.selectors.widget);
+        }
+
+        getDescription() {
+            return this.element.querySelector(Wizard.selectors.description);
+        }
+
+        getLabel() {
+            return this.element.querySelector(Wizard.selectors.label);
+        }
+
+        getErrorDiv() {
+            return this.element.querySelector(Wizard.selectors.errorDiv);
+        }
+
+        getTooltipDiv() {
+            return this.element.querySelector(Wizard.selectors.tooltipDiv);
+        }
+
+        getQuestionMarkDiv() {
+            return this.element.querySelector(Wizard.selectors.qm);
+        }
+
+        getPreviousButtonDiv() {
+            return this.element.querySelector(Wizard.selectors.previousButton);
+        }
+
+        getNextButtonDiv() {
+            return this.element.querySelector(Wizard.selectors.nextButton);
+        }
+
+
+        /**
+         * Binds Tabs event handling
+         *
+         * @private
+         */
+        #bindEvents() {
+            // var tabs = this._elements["tab"];
+            // if (tabs) {
+            //     for (var i = 0; i < tabs.length; i++) {
+            //         var _self = this;
+            //         (function(index){
+            //             tabs[index].addEventListener("click", function(event) {
+            //                 _self.#navigateAndFocusTab(index);
+            //             });
+            //         }(i));
+            //
+            //         tabs[i].addEventListener("keydown", function(event) {
+            //             _self.#onKeyDown(event);
+            //         });
+            //     };
+            // }
+            var _self = this;
+            this.getNextButtonDiv().addEventListener("click", function (event){
+                _self.#navigateToNextTab();
+            })
+            this.getPreviousButtonDiv().addEventListener("click", function (event){
+                _self.#navigateToPreviousTab();
+            })
+
+
+        }
+
+        /**
+         * Returns the index of the active tab, if no tab is active returns 0
+         *
+         * @param {Array} tabs Tab elements
+         * @returns {Number} Index of the active tab, 0 if none is active
+         */
+        getActiveIndex(tabs) {
+            if (tabs) {
+                for (var i = 0; i < tabs.length; i++) {
+                    if (tabs[i].classList.contains(Wizard.selectors.active.tab)) {
+                        return i;
+                    }
+                }
+            }
+            return 0;
+        }
+
+
+        #setActive(tabs) {
```

Comment:
Wizard is a series if steps. so, IMO, better to use `steps` terminology instead of `tabs`

Reply:
Hi @nit23uec, IMHO, tabs will be easier to understand for a developer because of the html of wizard. I also thought on the same before implementing using tabs term but using steps in code was not sounding correct.

Reply:
ack. thanks.